ta/gcov/include/user_ta_header_defines.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 NXP
+ */
+
+#ifndef USER_TA_HEADER_DEFINES_H
+#define USER_TA_HEADER_DEFINES_H
+
+#include "ta_gcov.h"
+
+#define TA_UUID TA_GCOV_UUID
+
+#define TA_FLAGS                (TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR | \
+				TA_FLAG_SECURE_DATA_PATH | \
+				TA_FLAG_CACHE_MAINTENANCE)
```

Comment:
These flags are not needed. `#define TA_FLAGS	0` should be fine here.

Reply:
yes, thanks